queries/scfg/highlights.scm

```diff
@@ -0,0 +1,30 @@
+; The MIT License (MIT)
+;
+; Copyright (c) 2023 Tim Culverhouse
+;
+; Permission is hereby granted, free of charge, to any person obtaining a copy
+; of this software and associated documentation files (the "Software"), to deal
+; in the Software without restriction, including without limitation the rights
+; to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+; copies of the Software, and to permit persons to whom the Software is
+; furnished to do so, subject to the following conditions:
+;
+; The above copyright notice and this permission notice shall be included in all
+; copies or substantial portions of the Software.
+;
+; THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+; IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+; FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+; AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+; LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+; OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+; SOFTWARE.
```

Comment:
That's a rather egregious license-to-content ratio. I'd prefer a clean room query. (Licenses should be compatible, in any case.)

Reply:
I guess the original author can confirm if they're okay with dropping the licence, but I can't remove it myself since it's not my code.

Reply:
Just write your own code, then. I'm sure you ca do better than two or three random captures.

Reply:
I'm perfectly fine removing the license, or @WhyNotHugo if you want to wholesale redo the queries or the parser I'm good with that.

Reply:
Thanks! I've removed the licence for now. I might try and improve upon this later, but, TBH, it works fine as-is so far.